app/src/test/java/com/group80/uoftinder/RecommendationProfileDisplayTest.java

```diff
@@ -0,0 +1,39 @@
+package com.group80.uoftinder;
```

Comment:
I know your tests are still being created but when you do get to finishing your test cases, don't forget to include documentation so it is easy to follow along and understand what is being tested. Thank you!

Reply:
Thanks for the reminder Nicholas!